validation-tool-backend/src/review/review.controller.ts

```diff
@@ -73,4 +73,26 @@ export class ReviewController {
     );
     return true;
   }
+
+  @Get('/progress')
+  @UseGuards(JwtAuthGuard)
+  async getReviewProgress(
+    @Req() req
+    ) {
+    const user: any = req.user;
+    const run_ids = user?.privileges.map((x) => x.run_id);
+    const result = await this.reviewService.getProgress(run_ids);
```

Comment:
for this and the statistics endpoint, we can avoid a database call if the user has no privileges on any run_id.

Reply:
done